pdns.c

```diff
@@ -39,17 +40,29 @@ present_text_lookup(pdns_tuple_ct tup,
 
 	/* Timestamps. */
 	if (tup->obj.time_first != NULL && tup->obj.time_last != NULL) {
+		char duration[50];
+
+		if (ns_format_ttl(tup->time_last - tup->time_first + 1, //non-0
```

Comment:
Is "non-0" supposed to be "no -0" ?  Like the "no +1" below

Reply:
please edit the commit summary when merging to change "version 2.2.0" to "version 2.2.1"